pdfminer/pdfdocument.py

```diff
@@ -4,11 +4,20 @@
 import struct
 
 try:
-    from Crypto.Cipher import ARC4, AES
-    from Crypto.Hash import SHA256
+    from cryptography.hazmat.primitives.ciphers import (Cipher, algorithms,
+                                                        modes)
+    from cryptography.hazmat.backends import default_backend
+    AES = True
 except ImportError:
-    AES = SHA256 = None
-    from . import arcfour as ARC4
+    AES = None
+
+try:
+    from hashlib import sha256
+except ImportError:
+    sha256 = None
```

Comment:
This try-except is not needed anymore

Reply:
I agree. I tried to change it line for line as little as possible as that tends to make many maintainers happy. Will remove

Reply:
Yep, will remove. I tried to change it line for line as little as possible because that makes many library maintainers happy